userguide/aws/tutorials/single-tier.adoc

```diff
@@ -0,0 +1,73 @@
+
+
+== Step 2: Subscribe to the Kill Bill AMI
+
+To start the installation process, point your browser to the Kill Bill AMI at
++++
+<a href="https://aws.amazon.com/marketplace/pp/B083LYVG9H?ref=_ptnr_doc_">
+AWS Marketplace
+</a>
++++.
+
+You should see the following image at the top of your screen:
+
+image::../../assets/aws/single-ami-subscribe.png[align=center]
+
+Click *Continue to Subscribe*. The next page will give the AWS Terms and Conditions. Accept the terms if asked. You will then see a new message confirming that you have subscribed. Next, click *Continue to Configuration*.
+
+== Step 3: Set Up EC2 Configuration
+
+The next page will give several configuration options.
+
+image::../../assets/aws/single-ami-configure.png[align=center]
+
+Be sure to select the correct region for your account. Accept the other defaults, and click *Continue to Launch*.
+
+The next page will give you several options for the launch method. Choose *Launch through EC2*.
+
+image::../../assets/aws/single-ami-launch.png[align=center]
+
+All other options will disappear. Click *Launch*.
+
+== Step 4: Create a Key Pair
+
+The next page is headed *Launch an Instance*. Scroll down to the middle of this page, to the box titled *Key Pair (login)* Here you are asked to choose or create a *key pair*.
+
+image::../../assets/aws/single-tier-keypair.png[align=center]
+
+We will create a new key pair. Click *Create Key Pair* to display a pane to be used for the creation. Give the key pair a simple, easy to remember name such as `My-Key-Pair`. Do not change the other options on this pane. Then click *Download Key Pair*. *Important:* You *must* save the private key that will be generated in this step. If you lose this key, you will *not* be able to login to your instance. In addition, you must set the protection for this file so it is readable by the owner only.
+
+== Step 5: Launch Your Instance
+
+When the key pair is generated, click *Launch Instances*. You should see the screen below:
+
+image::../../assets/aws/single-tier-launching.png[align=center]
+
+Your instance is finally launching! To follow what is happening on the EC2 Dashboard, scroll all the way down to the bottom, and click *View Instance* at the bottom right. This will take you to the *Instances* screen which is part of the EC2 Dashboard.
+
+image::../../assets/aws/single-ami-instances.png[align=center]
+
+
+In a short time, the *Instance State* for your instance should indicate *Running*. Select the checkbox to the left of your instance ID. An information pane should open below with details about your instance.
+
+== Step 6: Testing
+
+You can now try to login to Kaui from your browser using the URL https://INSTANCE_IP:443, where INSTANCE_IP is the IPV4 address for your instance, given on your dashboard as *Public IPV4 Address*. If all goes well, this should display the Kaui login screen. The browser may complain that your connection is not secure, but it should offer you a way to bypass the problem.
+
```

Comment:
I think adding URLs like `https://INSTANCE_IP:443` causes some issues with SEO and need to be bypassed. @FlowWriter, can you confirm?

Reply:
How should this be stated? These URLs are important.

Reply:
I think these can be escaped using a backslash character (`\`) but I'll let Mary confirm.

Reply:
It's not an SEO problem, but to prevent Asciidoctor from automatically creating a link from the URL, add a backslash character in front of the URL: https://docs.asciidoctor.org/asciidoc/latest/macros/autolinks/#escaping-urls-and-email-addresses